src/genworld_gui.cpp

```diff
@@ -183,6 +183,16 @@ static const NWidgetPart _nested_generate_landscape_widgets[] = {
 				NWidget(NWID_SPACER), SetFill(1, 1),
 			EndContainer(),
 		EndContainer(),
+		NWidget(NWID_SPACER), SetMinimalSize(0, 6), SetFill(1, 1),
+		/* NewGRF and AI/GS Settings */
+		NWidget(NWID_HORIZONTAL, NC_EQUALSIZE), SetPIP(10, 0, 10),
+			NWidget(WWT_PUSHTXTBTN, COLOUR_ORANGE, WID_GL_NEWGRF_BUTTON), SetMinimalSize(0, 24), SetDataTip(STR_MAPGEN_NEWGRF_SETTINGS, STR_NULL), SetFill(1, 0),
+			NWidget(WWT_PUSHTXTBTN, COLOUR_ORANGE, WID_GL_SCRIPT_BUTTON), SetMinimalSize(0, 24), SetDataTip(STR_MAPGEN_SCRIPT_SETTINGS, STR_NULL), SetFill(1, 0),
+			NWidget(WWT_PUSHTXTBTN, COLOUR_ORANGE, WID_GL_CONFIG_GS_BUTTON), SetMinimalSize(0, 24), SetDataTip(STR_MAPGEN_CONFIG_GS, STR_MAPGEN_CONFIG_GS_TOOLTIP), SetFill(1, 0),
```

Comment:
No, actually good catch. What happened was that I was originally reusing strings (I had set it to the same tooltip as the one on the title screen), but then realized it was better not to reuse strings, so I made it null because I intended to make a tooltip for it but got caught up and forgot. I'll correct that in the next FP